dnsaas/settings.py

```diff
@@ -178,7 +179,7 @@ def __getitem__(self, item):
     'DEFAULT_PAGINATION_CLASS':
         'rest_framework.pagination.LimitOffsetPagination',
     'DEFAULT_VERSIONING_CLASS':
-        'rest_framework.versioning.AcceptHeaderVersioning',
+        'rest_framework.versioning.NamespaceVersioning',
```

Comment:
imo we couldn't (for now) drop previous behaviour (header version) and we should create our own versioning system which would combine `AcceptHeaderVersioning` and `NamespaceVersioning` - you could specify either version in url or in header. Calling `/api/<resource>` should recognise version in header, but calling `/api/v<version>/<resource>` should rely on version in url. What do you think @xliiv ?